tldw_Server_API/tests/Evaluations/fixtures/llm_responses.py

```
@@ -156,7 +156,7 @@ def get_cached_response(cls, prompt_type: str, prompt_content: str = None) -> Di
         # Generate a deterministic response based on prompt content
         if prompt_content:
             # Use hash to generate consistent scores
-            hash_val = int(hashlib.md5(prompt_content.encode()).hexdigest()[:8], 16)
+            hash_val = int(hashlib.md5(prompt_content.encode()).hexdigest()[:8], 16)  # nosec B324
```

Ruff (0.15.1)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)